Tribler/vlc.py

```diff
@@ -49,7 +49,12 @@
 from inspect import getargspec
 
 __version__ = "N/A"
-build_date  = "Fri Jun 19 13:45:56 2015"
+build_date  = "Wed Sep 30 15:51:48 2015"
```

Comment:
@whirm They have a newer version here: https://github.com/oaubert/python-vlc/blob/master/generated/vlc.py
Built on Nov. 5.

Try it if this doesn't work.